downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,688 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake 2201.1.0
+permalink: /downloads/swan-lake-release-notes/2201.1.0/
+active: swan-lake-2201.1.0
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-1-0
+    - /downloads/swan-lake-release-notes/2201-1-0-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-1-0-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake 2201.1.0 
+
+<em>2201.1.0 (Swan Lake) is the first update of 2201.1.0 (Swan Lake), and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2022R2 version of the Language Specification.</em> 
+
+
+## Update Ballerina
+
+If you are already using Ballerina, use the [Ballerina Update tool](/learn/tooling-guide/cli-tools/update-tool/) to directly update to Swan Lake Beta6 by running the command below.
+
+> `bal dist pull 2201.1.0`
+
+## Install Ballerina
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
+### New features
+
+#### Support for the spread operator in the list constructor
+
+Introduced spread operator support for the list constructor expression.
+
+If the spread operator in a list constructor expression is `...x`, then, `x` is expected to be a list (i.e., an array or a tuple). All the member values of the list that result from evaluating `x` are included in the list value being constructed.
+
+```ballerina
+import ballerina/io;
+
+public function main() {
+    int[] a1 = [3, 4];
+    int[] v1 = [1, 2, ...a1];
+    io:println(v1); // [1,2,3,4]
+
+    int[2] a2 = [6, 7];
+    int[] v2 = [1, 2, ...a1, 5, ...a2];
+    io:println(v2); // [1,2,3,4,5,6,7]
+
+    [int, string] t1 = [5, "s"];
+    any[] v3 = [...t1, "x"];
+    io:println(v3); // [5,"s","x"]
+
+    [boolean, int...] t2 = [false, 4, 7];
+    [string, int, string, boolean, int...] v4 = ["x", ...t1, ...t2];
+    io:println(v4); // ["x",5,"s",false,4,7];
+
+    var v5 = [4, ...t1, ...a2];
+    io:println(v5); // [4,5,"s",6,7];
+}
+```
+
+The spread operator is not allowed with a variable-length list if the inherent type of the list being constructed has required members that are not guaranteed to have been provided a value.
+
+```ballerina
+public function main() {
+    [int, string...] t1 = [5, "s"];
+    [int, string, string...] v1 = [...t1]; // results in an error since a value is not guaranteed to have been provided for the second tuple member
+
+    [int, boolean, string, int...] t2 = [5, false, "w"];
+    [int, boolean, anydata...] v2 = [...t2, "x", "y"]; // works as all fixed tuple members are guaranteed to have been provided values
+}
+```
+
+#### Allow `int*float`, `float*int`,` int* decimal`, `decimal*int`, `float/int`, `decimal/int`, `float%int`, and `decimal%int` multiplicative expressions
```

Comment:
Instead of  having this  lengthy title, can we summarize the change in title and  add  all these cases into a paragraph?

Reply:
@MaryamZi @gimantha

Reply:
Shall we say `Allowed multiplicative expressions with operands`?

Reply:
Maybe something like

```
#### Support for multiplicative expressions with `int` and `float` or `decimal` operands
```

Basically one operand is int and the other is float or decimal.

Reply:
Fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/4402/commits/f2119b04fce522df005a62aa4bf83f4454c2f8da.